test/resources/billing.test.js

```diff
@@ -1,43 +1,86 @@
 /* eslint-disable func-names */
-import { expect } from 'chai';
+import {expect} from 'chai';
 
 import EasyPost from '../../src/easypost';
 import * as setupPolly from '../helpers/setup_polly';
+import {MockMiddleware, MockRequest, MockRequestMatchRule, MockRequestResponseInfo} from "../helpers/mocking";
+
+
+const middleware = (request) => {
+    return new MockMiddleware(request, [
+        new MockRequest(
+            new MockRequestMatchRule('POST', 'v2\\/bank_accounts\\/\\S*\\/charges$'),
+            new MockRequestResponseInfo(200, {})
+        ),
+        new MockRequest(
+            new MockRequestMatchRule('POST', 'v2\\/credit_cards\\/\\S*\\/charges$'),
+            new MockRequestResponseInfo(200, {})
+        ),
+        new MockRequest(
+            new MockRequestMatchRule('DELETE', 'v2\\/bank_accounts\\/\\S*$'),
+            new MockRequestResponseInfo(200, {})
+        ),
+        new MockRequest(
+            new MockRequestMatchRule('DELETE', 'v2\\/credit_cards\\/\\S*$'),
+            new MockRequestResponseInfo(200, {})
+        ),
+        new MockRequest(
+            new MockRequestMatchRule('GET', 'v2\\/payment_methods$'),
+            new MockRequestResponseInfo(200, {
+                "id": "summary_123",
+                "primary_payment_method": {
+                    "id": "card_123",
+                    "last4": "1234",
+                },
+                "secondary_payment_method": {
+                    "id": "bank_123",
+                    "bank_name": "Mock Bank",
+                }
+            })
+        )
+    ]);
+}
 
 describe('Billing Resource', function () {
-  setupPolly.startPolly();
+    setupPolly.startPolly();
```

Comment:
We can remove the setupPolly function here if we aren't recording any cassettes.